### ISSUE TITLE
Enhancement: Gloss Window - radio toggle changing default hs selection

### ISSUE THREAD

**Issue by gregorydimitriadis** [DCS-LCSR/SignStream3#13]
Toggle between Fingerspelled and Lexical Sign - changes the handshape from one-handed to two-handed (whereas the default for Lexical Sign should still be one-handed)

**Comment by blazetravis** [comment]
This bug is NOT fixed.

When changing from Fingerspelled to Lexical Sign in the Morph-Phon Info window, it still changes to two-handed (and copies all of the handshapes from the first hand to the newly added second hand).

![screen shot 2015-10-27 at 1 15 35 pm](https://cloud.githubusercontent.com/assets/13489587/10766213/db3a0f98-7cac-11e5-984e-ddea7dd0766a.png)

![screen shot 2015-10-27 at 1 08 56 pm](https://cloud.githubusercontent.com/assets/13489587/10766042/f4a43162-7cab-11e5-89d5-c56a10bb5fe7.png)

I was using the Rachel 59 file, utterance 22.

It should be also noted that when I pressed cancel after I changed to Lexical Sign and noticed the bug, the info box went away as it should. Then I shift+clicked the fingerspelled gloss again to display the morph-phon info window to test again. This time I was immediately met with this message:

![screen shot 2015-10-27 at 1 12 15 pm](https://cloud.githubusercontent.com/assets/13489587/10766125/604837a6-7cac-11e5-99cc-9962eb22ac56.png)

I pressed cancel, and SS3 displayed (***note the empty, but still defaulted Two-handed):

![screen shot 2015-10-27 at 1 14 31 pm](https://cloud.githubusercontent.com/assets/13489587/10766196/b6c7940a-7cac-11e5-92c6-675eb3c7c884.png)
